airflow-core/src/airflow/api_fastapi/auth/managers/base_auth_manager.py

```diff
@@ -423,133 +423,133 @@ def batch_is_authorized_variable(
         )
 
     @provide_session
-    def get_authorized_dag_ids(
+    def get_authorized_connections(
```

Comment:
There is no change in this, I only moved `get_authorized_dag_ids` and `filter_authorized_dag_ids` after `filter_authorized_connections`.